ui/app/AppLayouts/WalletV2/SettingsTab.qml

```diff
@@ -0,0 +1,163 @@
+import QtQuick 2.13
+import QtGraphicalEffects 1.13
+
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+
+import "../../../imports"
+import "../Profile/Sections"
+
+Item {
+    id: root
+
+    Column {
+        anchors.top:parent.top
+        leftPadding: 20
+        rightPadding: 20
+        width: parent.width
+        spacing: 12
+
+        StatusExpandableSettingsItem {
+            anchors.left: parent.left
+            anchors.leftMargin: 20
+            anchors.right: parent.right
+            anchors.rightMargin: 20
+
+            visible : (walletV2Model.accountsView.currentAccount.walletType !== Constants.seedWalletType) &&
+                      (walletV2Model.accountsView.currentAccount.walletType !== Constants.watchWalletType) &&
+                      (walletV2Model.accountsView.currentAccount.walletType !== Constants.keyWalletType)
+            expandable: false
+            icon.name: "seed-phrase"
+            primaryText: qsTr("Back up seed phrase")
+            secondaryText: qsTr("Back up your seed phrase now to secure this account")
+            button.text: qsTr("Back up seed phrase")
+            button.enabled: !profileModel.mnemonic.isBackedUp
+            button.onClicked: backupSeedModal.open()
+        }
+
+        StatusExpandableSettingsItem {
+            anchors.left: parent.left
+            anchors.leftMargin: 20
+            anchors.right: parent.right
+            anchors.rightMargin: 20
+
+            visible : walletV2Model.accountsView.currentAccount.walletType !== Constants.watchWalletType
+            expandable: true
+            icon.name: "secret"
+            primaryText: qsTr("Account signing phrase")
+            secondaryText: qsTr("View your signing phrase and ensure that you never get scammed")
+            expandableComponent: showSigningPhraseExpandableRegion
+        }
+
+        StatusExpandableSettingsItem {
+            anchors.left: parent.left
+            anchors.leftMargin: 20
+            anchors.right: parent.right
+            anchors.rightMargin: 20
+
+            visible : walletV2Model.accountsView.currentAccount.walletType === Constants.keyWalletType
+            expandable: true
+            icon.name: "seed-phrase"
+            primaryText: qsTr("View private key")
+            secondaryText: qsTr("Back up your seed phrase now to secure this account")
+            button.text: qsTr("View private key")
```

Comment:
Is the secondary text correct? It seems this text should be with: Back up seed phrase not with view private key?

Reply:
You are right. This text doesn't make sense will check with design and update it!

Reply:
I spoke to Carl and he mentioned this should be changes soon. He mentioned we can make it "view your seed phrase and ensure it's stored in a safe place" and it is available in 2 cases when account I imported using a PK and using a Seed Phrase